frontend/packages/pipelines-plugin/src/components/quicksearch/PipelineQuickSearchVersionDropdown.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { Dropdown, DropdownItem, DropdownToggle } from '@patternfly/react-core';
+import { CheckCircleIcon } from '@patternfly/react-icons';
+import { global_palette_green_500 as greenColor } from '@patternfly/react-tokens/dist/js/global_palette_green_500';
+import { CatalogItem } from 'packages/console-dynamic-plugin-sdk/src';
+import { isSelectedVersionInstalled } from './pipeline-quicksearch-utils';
+
+interface PipelineQuickSearchVersionDropdownProps {
+  selectedVersion: string;
+  item: CatalogItem;
+  onChange: (key: string) => void;
+}
+
+const PipelineQuickSearchVersionDropdown: React.FC<PipelineQuickSearchVersionDropdownProps> = ({
+  item,
+  onChange,
+  selectedVersion,
+}) => {
+  const [isOpen, setOpen] = React.useState(false);
+  const toggleIsOpen = React.useCallback(() => setOpen((v) => !v), []);
+  const versions = item?.attributes?.versions ?? [];
+  const versionItems = versions.reduce((acc, { version, id }) => {
+    acc[id.toString()] = id === item.data?.latestVersion?.id ? `${version} (latest)` : version;
+    return acc;
+  }, {});
+
+  const getDropdownItem = React.useCallback(
```

Comment:
Not sure you need to use a callback here. You're not passing this as a prop thus you won't need to worry if it remounts on itself. Was this in reaction to something or was this proactive? I'm thinking it doesn't actually do anything making this a `useCallback`.